packages/slice-machine/src/legacy/lib/models/common/widgets/Group/ListItem/index.jsx

```diff
@@ -171,15 +170,35 @@ export const CustomListItem = ({
         {...rest}
         HintElement={HintElement}
         CustomEditElements={[
-          <Button
-            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-member-access
-            key={`custom-edit-element-${groupItem.key}`}
-            mr={2}
-            variant="buttons.darkSmall"
-            onClick={() => setSelectMode(true)}
-          >
-            Add Field
-          </Button>,
+          <AddFieldDropdown
+            key="add-field-dropdown"
+            disabled={newFieldData !== null}
+            onSelectField={onSelectFieldType}
+            fields={
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```

Comment:
Since we have a `eslint-disable` on top, I guess you can avoid disabled the rules here (an below), no? 👀

Reply:
Didn't even see the global disabled rule, I copy-paste the old code. Good catch.

Hum, it could be a good thing to do for sure, even if we would need to understand what would the consequence for us. Sometimes we are working on JS files still and we may want to have some protection. WDYT?

Reply:
Done 🙂

Reply:
I agree, we could keep the general rules but disable the TS ones for JS files to reduce the noise of all the disabled comments in these legacy components.